app/src/components/ExportPane.js

```diff
@@ -32,61 +36,163 @@ const getFormattedResult = (category, filteredResults) => {
-  const filteredResults = useMemo(() => allResults.filter((x) => !excludedItems.includes(x.id)), [
-    allResults,
-    excludedItems,
-  ]);
+  const [sections, setSections] = React.useState(() =>
+    getExportSections({ excludedItems, resultChunks, soapSummary, transcriptChunks }),
+  );
+  const [lang, setLang] = React.useState('en');
+  const [isLoadingTranslation, setIsLoadingTranslation] = React.useState(false);
+
+  React.useEffect(() => {
+    if (!visible) return;
+
+    const englishSections = getExportSections({ excludedItems, resultChunks, soapSummary, transcriptChunks });
+
+    if (lang === 'en') {
+      setSections(englishSections);
+      return;
+    }
+
+    setIsLoadingTranslation(true);
+
+    Auth.currentSession()
+      .then((session) => session.getIdToken().getJwtToken())
+      .then((token) => translateSections(englishSections, lang, token))
+      .then(setSections)
+      .finally(() => setIsLoadingTranslation(false));
+  }, [visible, excludedItems, resultChunks, soapSummary, transcriptChunks, lang]);
+
   return (
     <div className={cs(s.base, visible && s.visible)}>
-      <div className={s.page}>
-        <header>
-          <div className={s.logo} />
-        </header>
-
-        <main>
-          <ExportPaneHeader content='Summary' type='SECTION' />
-          <p>
-            Thank you for visiting the clinic today, {new Date().toISOString().slice(0, 10)}. Please take a moment to
-            review the following important information from today's consultation and reach out to us at +12345678910 if
-            you have any questions.
-          </p>
-          <p>{soapSummary}</p>
-
-          <ExportPaneHeader content='Medications' type='SUB_SECTION' />
-
-          <CategorySummary results={filteredResults} category='MEDICATION' />
-
-          <ExportPaneHeader content='Anatomy' type='SUB_SECTION' />
-          <CategorySummary results={filteredResults} category='ANATOMY' />
-
-          <ExportPaneHeader content='Medical Conditions' type='SUB_SECTION' />
-          <CategorySummary results={filteredResults} category='MEDICAL_CONDITION' />
-
-          <ExportPaneHeader content='Tests, Treatments, Procedures' type='SUB_SECTION' />
-          <CategorySummary results={filteredResults} category='TEST_TREATMENT_PROCEDURE' />
-
-          <ExportPaneHeader content='Visit Transcription' type='SECTION' />
-          <div className={s.transcript}>
-            <p>Below is the transcription for your visit -</p>
-            {(transcriptChunks ? transcriptChunks : []).map((t, i) => (
-              <p key={i}>{t.text}</p>
-            ))}
-          </div>
-        </main>
+      <div className={s.pageContainer}>
+        <Flex justifyContent='space-between' alignItems='center' bg='#ececec' px={4} py={2}>
+          <Heading as='div' size='md'>
+            Translate Summary
+          </Heading>
+          <SupportedLanguagesMenu currentLang={lang} onLangClick={setLang} disabled={isLoadingTranslation} />
```

Comment:
Can we a easily distinguishable boundary around the Menu to separate it from the page element?